chia/wallet/util/wallet_types.py

```diff
@@ -13,3 +17,8 @@ class WalletType(IntEnum):
     RECOVERABLE = 7
     DISTRIBUTED_ID = 8
     POOLING_WALLET = 9
+
+
+class AmountWithPuzzlehash(TypedDict):
+    amount: int
```

Comment:
normally we use `uint64` for amounts

Reply:
It looks like the existing values were hinted as `uint64`.  Updated.  Thanks for catching this.